dev/cnf/resources/bnd/bundle.props

```diff
@@ -52,3 +52,6 @@ Import-Package: ${defaultPackageImport}
 #require explicitly specifying all ds and metatype annotated classes.
 -dsannotations:
 -metatypeannotations:
+
+# With bnd 5.2.0 The importing of java.* packages need to be disabled to maintain legacy behavior.
```

Comment:
Why do we need to disable importing the java.* packages?

Reply:
I disabled importing java.* packages because they are not needed for the bundle to run. Some bundles use `*` for their Import-Package to find all packages from the `-buildpath`, which ends up including java.* now regardless of `-noimportjava: true` being set. So where bundles use `*`, I added a `!java.*` line to prevent those from importing java.* packages.

If I didn't remove the java.* imports, then the bundles would fail to start on server start. For example,
```
junit.framework.AssertionFailedError: Found issues in console.log after start: [[ERROR   ] CWWKE0702E: Could not resolve module: com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.hc.3.2 [194]
```

Reply:
Thanks for taking the time to explain that Ethan.